### PR TITLE
tweak currency list spacing in mobile send

### PR DESCRIPTION
**Pull request by zanderz**
shows the last 2 currency items again.

Couldn't repro the cut-off text in the receive dialog.

### CODE REVIEW

File: shared/wallets/send-form/choose-asset/index.js [hunk 1]
```diff
@@ -328,6 +328,9 @@ const styles = styleSheetCreate({
     isElectron: {
       maxHeight: 525 - 48,
     },
+    isMobile: {
+      paddingBottom: 100,
```
Comment (chrisnojima) [inline shared/wallets/send-form/choose-asset/index.js]: i don't think this is likely the correct fix

Comment (zanderz) [reply shared/wallets/send-form/choose-asset/index.js]: What do you have in mind?

Comment (chrisnojima) [reply shared/wallets/send-form/choose-asset/index.js]: this kind of styling fix typically is fixing the symptom and not the actual problem. likely this list isn't setting its height correctly so its layout is wrong.

**Comment by zanderz** [comment]
@keybase/react-hackers Changed to use flow values instead of some absolute percentage. Tried ios and android sims.

**Pull request merged**